main/lsp/lsp_helpers.cc

```diff
@@ -214,7 +214,9 @@ string prettyTypeForConstant(const core::GlobalState &gs, core::SymbolRef consta
     ENFORCE(constant == constant.dealias(gs));
 
     core::TypePtr result;
-    if (constant.isClassOrModule()) {
+    if (constant == core::Symbols::StubModule()) {
+        result = core::Types::untyped(gs, constant);
```

Comment:
Maybe we should just return a string that says "this constant doesn't exist"?

Reply:
@elliottt pointed out that `T.reveal_type` on a constant like this returns `T.untyped`, so there is at least some justification for untyped here.

I don't have strong opinions; I guess "this thing doesn't exist" is more user-friendly than `T.untyped`?

Reply:
I'm fine with either, both are more informative than reporting the location of `StubModule` :)